migration-engine/migration-engine-tests/tests/create_migration/create_migration_tests.rs

```diff
@@ -982,8 +982,7 @@ fn alter_constraint_name(mut api: TestApi) {
         .assert_migration_directories_count(2)
         .assert_migration("custom",move |migration| {
             let expected_script = if is_mssql {
-                indoc! {
-                     r#"
+                expect![[r#"
```

Comment:
Did you try what happens if you make the expect to break, and then do `env UPDATE_EXPECT=1 cargo test`?

Reply:
Yep, it works.